spec.html

```diff
@@ -9201,7 +9201,7 @@ <h1>[[DefineOwnProperty]] ( _P_, _Desc_ )</h1>
               1. If _Desc_ has a [[Writable]] field and if _Desc_.[[Writable]] is *false*, return *false*.
               1. If _Desc_ has a [[Value]] field, then
                 1. Let _value_ be _Desc_.[[Value]].
-                1. Return ? IntegerIndexedElementSet(_O_, _numericIndex_, _value_).
+                1. Return ! IntegerIndexedElementSet(_O_, _numericIndex_, _value_).
```

Comment:
Still fallible b/c of ToNumber/ToBigInt.

Reply:
Oops, yes, my bad. 🤦